spec/integration/outputs/templates_spec.rb

```diff
@@ -73,7 +73,7 @@
   end
 
   it "make [geoip][location] a geo_point" do
-    results = @es.search(:body => { "filter" => { "geo_distance" => { "distance" => "1000km", "geoip.location" => { "lat" => 0.5, "lon" => 0.5 } } } })
+    results = @es.search(:body => { "query" => { "bool" => { "must" => { "match_all" => {} }, "filter" => { "geo_distance" => { "distance" => "1000km", "geoip.location" => { "lat" => 0.5, "lon" => 0.5 } } } } } })
```

Comment:
you could just do:
`results = @es.search(:body => { "query" => { "geo_distance" => { "distance" => "1000km", "geoip.location" => { "lat" => 0.5, "lon" => 0.5 } } } })` (replacing `filter` with `query`)